content/02.delete-me.md

```diff
@@ -1,15 +1,146 @@
+
+Horizontal rule:
+
+---
+
+A long paragraph of text.
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+
+Another long paragraph.
+Donec massa sapien faucibus et molestie ac feugiat sed lectus.
+Est sit amet facilisis magna etiam.
+Ornare arcu dui vivamus arcu felis bibendum.
+Magna fermentum iaculis eu non diam.
+Vehicula ipsum a arcu cursus vitae.
+Ante metus dictum at tempor commodo ullamcorper a.
+Vitae tortor condimentum lacinia quis vel eros.
+Ut venenatis tellus in metus vulputate eu scelerisque felis imperdiet.
+Facilisis sed odio morbi quis.
+Enim ut sem viverra aliquet eget sit amet tellus.
+
+*Italic* _text_
+
+**Bold** __text__
+
+Combined *italics and __bold__*
+
+~~Strikethrough~~
+
+Line break without starting new paragraph by putting  
+two spaces at end of line
+
+1. Ordered list item
+2. Ordered list item
+    a. Sub-item
+    b. Sub-item
+        i. Sub-sub-item
+3. Ordered list item
+    a. Sub-item
+
+* List item
+- List item
++ List item
+
+superscript<sup>2</sup>
+
+subscript<sub>2n+1</sub>
+
+[Link](https://www.google.com)
+
+[Link with title (hover) text](https://www.google.com "Google")
+
+[Link by reference][google homepage]
+
+[Google Homepage]: https://www.google.com
+
+[Link styled as a button](https://www.google.com "Google"){.button}
+
+DOI citation [@doi:10.15363/thinklab.4]
+
+URL citation [@url:https://greenelab.github.io/meta-review]
+
+Citation by tag [@tag:deep-review]
+
+> Quoted text 
+
+> Quoted block of text  
+> 
+> Two roads diverged in a wood, and I—  
+> I took the one less traveled by,  
+> And that has made all the difference.  
+
+Code `in the middle` of normal text, aka `inline code`
+
+```c++
+// code block with C++ syntax highlighting
+int main() {
+    // a string too long to fit within page
+    cout << "It was the best of times, it was the worst of times, it was the age of wisdom, it was the age of foolishness";
+    return 0;
+}
+```
+
+```
+// code block with no syntax highlighting
+int count = 0;
+function add() {
+	count++;
+}
+```
+
+![A square image at actual size and with a bottom caption](https://raw.githubusercontent.com/greenelab/manubot-resources/master/images/square.png "Square image"){#fig:square-image}
+
+![An image too wide to fit within page at full size](https://raw.githubusercontent.com/greenelab/manubot-resources/master/images/wide.png "Wide image"){#fig:wide-image}
+
+![A tall image with a specified height](https://raw.githubusercontent.com/greenelab/manubot-resources/master/images/tall.png "Tall image"){#fig:tall-image height=3in}
```

Comment:
We will want to reference all three images in the main text somewhere.

Reply:
Let's also get an example of embedding an SVG from GitHub using `?sanitize=true`